### DISCUSSION TITLE
Save user settings based on user account

### DISCUSSION THREAD

**Discussion by shinobu-uwu** [ppy/osu#21898] [Ideas] [answered]
We could have an option for the user to upload and sync his settings based on his account. This is something I find very useful in league of legends, every time I play a game on a fresh install it automatically syncs my interface and keybindings preferences.

Now that we can customize pretty much everything we see during gameplay being able to automatically sync to what you had previously would be a god sent especially to LAN tournament players, since I find the process of remaking your layout tedious and error prone.

Of course not all settings should be synced, graphics and audio shouldn't for example, but interface, keybinding preferences and in-game layout should IMO. This will also require changes to [ppy/osu-web](https://github.com/ppy/osu-web)

**Comment by peppy** [comment, marked as answer]
We'll be adding settings presets and potentially have the ability to save against your account in the future. But this is a distant future.
